int32 id
string type
string default_model
spatial_db/Point2DModel[] point2d_models
spatial_db/Pose2DModel[] pose2d_models
spatial_db/Polygon2DModel[] polygon2d_models
spatial_db/Point3DModel[] point3d_models
spatial_db/Pose3DModel[] pose3d_models
spatial_db/Polygon3DModel[] polygon3d_models
spatial_db/TriangleMesh3DModel[] trianglemesh3d_models
spatial_db/PolygonMesh3DModel[] polygonmesh3d_models
